testing/src/main/java/org/oppia/android/testing/TestMediaPlayer.kt

```diff
@@ -0,0 +1,9 @@
+package org.oppia.android.testing
```

Comment:
Let's avoid over-populating the top-level directory. I suggest: ``org.oppia.android.testing.media``.

Reply:
Done

Reply:
resolving on behalf of Ben